src/MovieCard.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 import './MovieCard.css'
+import {NavLink} from 'react-router-dom'
 
-const MovieCard = ({ id, image, title, selectMovie}) => {
+const MovieCard = ({ id, image, title}) => {
     return (
-      <section onClick={() => selectMovie(id)}className='movie-card'>
```

Comment:
Adding the NavLink to the movie cards makes alot of sense rather than using an OnClick to change page views